lib/qrstorage_web/controllers/qr_code_controller.ex

```diff
@@ -28,6 +28,7 @@ defmodule QrstorageWeb.QrCodeController do
         end
 
         conn
+        |> put_flash(:admin_url_id, qr_code.admin_url_id)
```

Comment:
I understand why you put the `admin_url_id` in the flash section. However, I think including this as a param in the redirect feels cleaner instead using the workaround via `put_flash(...)`